zproject/settings.py

```diff
@@ -858,7 +858,7 @@ def get_secret(key):
     },
     'stats': {
         'source_filenames': (
-            'node_modules/plotly.js/dist/plotly.js',
+            'node_modules/plotly.js/dist/plotly.min.js',
             'node_modules/jquery/dist/jquery.js',
             'js/portico/stats.js'
         ),
```

Comment:
I would move MINIFIED_JS to be an option on this `stats` object, rather than a global.  Specifically, I'm thinking we add a `minified_source_filenames` parameter that is its own entry, and we just concatenate those as part of the overall javascript.  

May need to also patch `zerver/templatetags/minified_js.py` so it works in development.